File: SHA-3/Java/main.java
public class main {

    public static void main(String[] args) {
        int length = 256;
        String plainText = "Furtsy denemee";
        System.out.println("Şifrelenmemiş hali: " + plainText);
        SHA3 sha3Object = new SHA3();
        String digest = sha3Object.digest(plainText, length);
        System.out.println("SHA3 ile şifrelenmiş hali: \n" + digest);
    }
}

File: SHA-3/Js/main.js
const crypto = require('crypto');

class SHA3 {
  digest(text, size) {
    const digestObject = crypto.createHash(`sha3-${size}`);
    digestObject.update(text, 'utf8');
    return digestObject.digest('hex');
  }
}

const length = 256;
const plainText = 'Furtsy denemee';

console.log(`Metnin şifrelenmemiş hali: ${plainText}`);

const sha3Object = new SHA3();
const digest = sha3Object.digest(plainText, length);

console.log(`SHA3 ile şifrelenmiş hali:\n${digest}`);